packages/astro/src/core/render/dev/index.ts

```diff
@@ -79,7 +79,7 @@ export async function render(renderers: Renderer[], mod: ComponentInstance, ssrO
 				links.add({
 					props: {
 						rel: 'stylesheet',
-						href,
+						href: `${href}?t=${Date.now()}`,
```

Comment:
In firefox, it felt like HMR was broken for tailwind styles but it turns out the browser is just ignoring the `<link>` update because the href is unchanged

(this should probably be cleaned up to handle existing query params in the href just in case)